CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+### Date:       2024-March-29
+### Release:    v2024032901
+
+### Export Buttons Fixed
```

Comment:
A little nit picky, but there should be 4 (####) here. It's so that the md file gets formatted correctly and the changes have a different heading than the date and the release version